CMakeLists.txt

```diff
@@ -45,6 +45,8 @@ add_definitions (
     -DBOOST_IOSTREAMS_DYN_LINK
     -DBOOST_PROGRAM_OPTIONS_DYN_LINK
     -DBOOST_ALL_NO_LIB)
+else()
+    add_definitions(-fPIC -g)
```

Comment:
@edenhill Should I add `-O2` here?

Reply:
Unlewss -O2 is set somewhere else (some base CMakefiles perhaps?) it does seem like a good idea, yeah.

Reply:
When I build things I see the c++/gcc commands being ran, and I didn't see `-O2` in it anywhere. I'll add it in as I wouldn't want `-g` debug symbols to affect any gcc optimizations it can make.